packages/ui/src/working-groups/modals/ApplicationDetailsModal/FormDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import {
+  SidePaneColumn,
+  SidePaneLabel,
+  SidePaneRow,
+  SidePaneTable,
+  SidePaneText,
+} from '../../../common/components/SidePane'
+import { useApplicationQuestionAnswers } from '../../hooks/useApplicationQuestionAnswers'
+import { ApplicationFormQuestionAnswer } from '../../types/ApplicationFormQuestionAnswer'
+
+interface Props {
+  applicationId: string
+}
+
+export const FormDetails = React.memo(({ applicationId }: Props) => {
+  const { isLoading, answers } = useApplicationQuestionAnswers(applicationId)
+
+  if (isLoading)
+    return (
+      <SidePaneTable>
+        <SidePaneText>Loading...</SidePaneText>
+      </SidePaneTable>
+    )
+
+  return (
+    <SidePaneTable>
+      {answers?.map((answer) => (
+        <QuestionAnswerPair answer={answer} key={JSON.stringify(answer)} />
```

Comment:
Maybe a better would be `question.index` here instead of `JSON.stringify()`. The `index` should uniquely identify questions from the set so it falls into `key` constraints. The other thing is that, although negligible, there is a cost for JSON.stringify compared to property access.

Reply:
I'm wondering if we shouldn't also sort them by index before rendering, to make sure they show up in the same order as  they appeared on the form

Reply:
Yep, we can do that as well. I've done that in the application form for questions.